### PR TITLE
Create /appliance/lxd pages

### LINKED ISSUE #8356: Create /appliance/lxd pages
[Brief doc](https://docs.google.com/document/d/1uU1XRM6vYLkB1LwLU9uU1FleJVpydvDGAODL10RdMng/edit?ts=5f50ca45)

Create:

- [ ] /appliance/lxd
- [ ] /appliance/lxd/raspbery-pi
- [ ] /appliance/lxd/intel-nuc

Do not add this to the other bits of the bubble as it is a beta

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Create /appliance/lxd pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliance/lxd
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page only provides VM and Raspberry pi 4 instructions (lots of logic and variables for this but apparently it will be reused in the future)
- Refer to the [brief](https://docs.google.com/document/d/1uU1XRM6vYLkB1LwLU9uU1FleJVpydvDGAODL10RdMng/edit?ts=5f50ca45) for copy and yaml

## Issue / Card

Fixes #8356

### CODE REVIEW

**Comment by anthonydillon** [comment]
Should there be a link from the /appliance/portfolio page?

File: templates/appliance/lxd/index.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      link: "https://snapcraft.io/lxd"
+      publisher: "Canonical"
+      channel: "latest/stable"
+      version: "4.5"
+      published_date: "18 September 2020"
+---
+
+#### System container manager and API
+
+LXD is a system container manager
```
Comment (anthonydillon) [inline templates/appliance/lxd/index.md:52]: It looks like these short sentences without full stops should be headings?

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Code looks good but the bottom of the LXD page is missing "Try it out!" section

File: templates/appliance/lxd/index.md (hunk 1 above)
Comment (pmahnke) [reply templates/appliance/lxd/index.md:52]: fixed

**Comment by pmahnke** [comment]
It should not yet be on the homepage.

Also, I see the next steps... 

image

**Comment by anthonydillon** [comment]
The section I mean it's at the bottom. It's missing the title and copy in the left:

**Comment by pmahnke** [comment]
@anthonydillon fixed the 'Try it out section'

image

**Comment by pmahnke** [comment]
@anthonydillon ready for another look

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

**Pull request merged**